tchannel/tornado/connection.py

```diff
@@ -49,6 +51,13 @@
 log = logging.getLogger('tchannel')
 
 
+#: Sentinel object representing that the connection is outgoing.
+OUTGOING = object()
```

Comment:
Yes, that's also valid. Why do you think it's better? It takes away some of the safety of sentinel values.